modules/dpu-modify-crio-logging-.adoc

```diff
@@ -0,0 +1,393 @@
++
+[source,terminal]
+----
+$ oc label node <NODENAME> node-role.kubernetes.io/dpu-worker=
+----
+
+. Label the DPU nodes with node-role:
++
+[source,terminal]
+----
+$ oc label node <NODENAME> node-role.kubernetes.io/dpu-host=
+----
+
+. Create a Cluster Network Operator (CNO) ConfigMap in the infrastructure cluster setting the mode to DPU. Save the YAML in the `ovnkubeconfigmap.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: dpu-mode-config
+    namespace: openshift-network-operator
+data:
+    mode: "dpu"
+immutable: true
+----
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc create -f ovnkubeconfigmap.yaml
+----
+
+. Label the DPU nodes in the infrastructure cluster where you want to enable hardware offloading.
++
+[source,terminal]
+----
+$ oc label node <NODENAME> network.operator.openshift.io/dpu=
+----
+
+== Configuring support for hardware offloading in the tenant cluster
+
+Configure support for hardware offloading in the tenant cluster by using this procedure.
+
+
+.Procedure
+
+. Create a MachineConfigPool for all the DPU workers. Save the YAML in the `dputenantmachineconfig.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: dpu-host
+spec:
+  machineConfigSelector:
+    matchExpressions:
+    - key: machineconfiguration.openshift.io/role
+      operator: In
+      values:
+      - worker
+      - dpu-host
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/dpu-host: ""
+----
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc create -f dputenantmachineconfig.yaml
+----
+
+. Label the DPU nodes:
++
+[source,terminal]
+----
+$ oc label node <NODENAME> node-role.kubernetes.io/dpu-host=
+----
+
+. Install and configure the SR-IOV Network Operator:
+
+This procedure describes how to install the SR-IOV Network Operator using the web console. However, if the console is not reachable due to the ingress being down follow the guidance on "CLI: Installing the SR-IOV Network Operator".
+
+.. In the {product-title} web console, click *Administration* -> *Namespaces*.
+
+.. Click *Create Namespace*.
+
+.. In the *Name* field, enter `openshift-sriov-network-operator`, and click *Create*.
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Select *SR-IOV Network Operator* from the list of available Operators, and click *Install*.
+
+.. On the *Install Operator* page, under *A specific namespace on the cluster*, select *openshift-sriov-network-operator*.
+
+.. Click *Install*.
+
+.. Verify that the SR-IOV Network Operator is installed successfully:
+
+... Navigate to the *Operators* -> *Installed Operators* page.
+
+... Ensure that *SR-IOV Network Operator* is listed in the *openshift-sriov-network-operator* project with a *Status* of *InstallSucceeded*.
```

Comment:
Can we just point them to the existing sriov docs?

Reply:
The rest looks good, thanks

Reply:
This section is not used in the content. I think I added by mistake, I will remove this file